application/classes/Ushahidi/Console/PostExporter.php

```diff
@@ -131,34 +131,46 @@ protected function executeExport(InputInterface $input, OutputInterface $output)
 			if ($job->filters) {
 				$filters = array_merge($filters, $job->filters);
 			}
-			
+
 			// Set the fields that should be included if set
 			if ($job->fields) {
 				$data->include_attributes = $job->fields;
 			}
 		}
 
-        foreach ($filters as $key => $filter) {
-            $data->$key = $filter;
+		foreach ($filters as $key => $filter) {
+			$data->$key = $filter;
+		}
+		$this->postExportRepository->setSearchParams($data);
+		$posts = $this->postExportRepository->getSearchResults();
+		service("formatter.entity.post.$format")->setFileSystem($this->fs);
+		service("formatter.entity.post.$format")->setAddHeader($add_header ===  'true');
```

Comment:
is there a way to make the input from the cli be an expected type? (to get a bool instead of a "true" string)